pkgs/development/python-modules/openai-triton/default.nix

```diff
@@ -34,35 +36,32 @@ let
   # pkgsTargetTarget maybe doesn't matter, because ptxas compiles programs to
   # be executed on the GPU.
   # Cf. https://nixos.org/manual/nixpkgs/unstable/#sec-cross-infra
-  ptxas = "${pkgsTargetTarget.cudaPackages.cuda_nvcc}/bin/ptxas"; # Make sure cudaPackages is the right version each update (See python/setup.py)
+  ptxas = "${cudaPackages.cuda_nvcc}/bin/ptxas"; # Make sure cudaPackages is the right version each update (See python/setup.py)
```

Comment:
Is the comment above still relevant now that you removed `pkgsTargetTarget` ?

Reply:
Why are we removing ``pkgsTargetTarget``?